src/python/grpcio_tests/tests/unit/credentials/cert_hier_1/certs/ca.cert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Even though they're just test data, how would you feel about adding a `README.md` that describes what these files are and what they mean to one another?

Reply:
yep. adding a `README.md`

Reply:
done. added description to existing `README.md` (renamed from `README`).